xla/service/gpu/gpu_windowed_einsum_handler_test.cc

```diff
@@ -287,6 +296,71 @@ ENTRY main.12_spmd {
                       m::Op(), m::Op(), m::Op(), m::Op()),
                   m::Op())));
 }
+TEST_F(GpuWindowedEinsumHanlderTest, A2aGemmHaveStreamIds) {
+  constexpr absl::string_view kHloString = R"(
+HloModule pjit__unnamed_wrapped_function_, entry_computation_layout={(bf16[1,8192,32768]{2,1,0}, bf16[1,4,2048,8192]{3,2,1,0})->bf16[1,4,2048,32768]{3,2,1,0}}, num_partitions=8
+
+ENTRY main.9_spmd {
```

Comment:
Can you add filecheck tests. I think it would be easier to follow the transform then.

Reply:
updated to use filecheck.